filters/accesslog/control_test.go

```diff
@@ -41,34 +42,44 @@ func TestAccessLogControl(t *testing.T) {
 			msg:     "disables-access-log",
 			state:   NewDisableAccessLog(),
 			args:    nil,
-			result:  AccessLogFilter{false, make([]int, 0)},
+			result:  AccessLogFilter{Enable: false, Prefixes: make([]int, 0)},
 			isError: false,
 		},
 		{
 			msg:     "disables-access-log-selective",
 			state:   NewDisableAccessLog(),
 			args:    []interface{}{1, 201, 30},
-			result:  AccessLogFilter{false, []int{1, 201, 30}},
+			result:  AccessLogFilter{Enable: false, Prefixes: []int{1, 201, 30}},
 			isError: false,
 		},
 		{
 			msg:     "disables-access-log-convert-float",
 			state:   NewDisableAccessLog(),
 			args:    []interface{}{1.0, 201},
-			result:  AccessLogFilter{false, []int{1, 201}},
+			result:  AccessLogFilter{Enable: false, Prefixes: []int{1, 201}},
 			isError: false,
 		},
+		{
+			msg:     "mask-access-log-query",
+			state:   NewMaskAccessLogQuery(),
+			args:    []interface{}{"key_1"},
+			result:  AccessLogFilter{Enable: true, MaskedQueryParams: map[string]bool{"key_1": true}},
+			isError: false,
+		},
+		{
+			msg:     "mask-access-log-query-convert-int",
+			state:   NewMaskAccessLogQuery(),
+			args:    []interface{}{1},
+			result:  AccessLogFilter{},
+			isError: true,
+		},
 	} {
 		t.Run(ti.msg, func(t *testing.T) {
 			f, err := ti.state.CreateFilter(ti.args)
 
 			if ti.isError {
-				if err == nil {
-					t.Errorf("Unexpected error creating filter %v", err)
-					return
-				} else {
-					return
-				}
+				require.Error(t, err, "Expected error creating filter")
```

Comment:
minor: you can also use `require.Condition` to remove the outer if.

Reply:
I guess @MustafaSaber meant to remove the `if ti.isError` and replace it with a oneliner err check.
I don't like require/assert lib but feel free to let or change it.